### ISSUE TITLE
Are there any plans to include javadoc?

### ISSUE THREAD

**Issue by jaredrummler** [CloudRail/cloudrail-si-android-sdk#76]
Adding the javadoc.jar to maven central would be helpful.

**Comment by PatrickStoklasa** [comment]
I agree, this should be part of the release. We actually have this on our roadmap already. I will let you know as soon as it is included.
